cfunits-python/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+$PYTHON setup.py install --record record.txt
```

Comment:
``cfunits-python`` does not use ``setuptools``. I sent them a PR for it some time ago, but they want to stick with ``distutils``.